Library/OcMainLib/OpenCoreMisc.c

```diff
@@ -930,6 +930,8 @@ OcMiscBoot (
              );
   if (!EFI_ERROR (Status)) {
     Context->PickerCommand = OcPickerBootAppleRecovery;
+  } else if (OcIsAppleHibernateWake () && Config->Misc.Boot.HidePickerHibernate) {
+    Context->PickerCommand = FALSE;
```

Comment:
Hmmm, but what if hibernate boot entry was not found?

Reply:
@vit9696 This is at the early stage where there's just information from the config parsed and no actual handling has taken place yet. Thus, the idea of this looks just fine. If the hibernate boot entry is not found, it should invoke the same handling we already have in place for when `ShowPicker` is false.

@khronokernel However, this is still broken:
1. ``FALSE`` is not a valid value for ``PickerCommand``.
2. As `OcIsAppleHibernateWake()` performs "actual work", it should be on the right-hand side.
3. This is special handling for when `ShowPicker` is enabled. Thus, either the conditions could be appended to the `ShowPicker` handling as such:
```
else if (Config->Misc.Boot.ShowPicker && (!Config->Misc.Boot.HidePickerHibernate || !OcIsAppleHibernateWake ())) {
  Context->PickerCommand = OcPickerShowPicker;
}
```
... or it could be handled more explicitly inside:
```
else if (Config->Misc.Boot.ShowPicker) {
  if (!Config->Misc.Boot.HidePickerHibernate || !OcIsAppleHibernateWake ()) {
    Context->PickerCommand = OcPickerShowPicker;
  } else {
    Context->PickerCommand = OcPickerDefault;
  }
}
```
5. There is no concept for the interactive side of things. Say a macOS installation is hibernated and the user manually enters the picker with no memory of hibernating, then this option isn't of much help. I think Apple completely blocks entering the picker, though that seems tricky regarding potential boot loops. This could be solved in many ways, such as a special hotkey to force-enter the picker, or at least having a visual indicator of hibernation, such as in the boot entry title and icon. @vit9696 Preferences?

Reply:
@vit9696 @khronokernel Actually, if this remains to be a special mutator for `ShowPicker`, maybe it should be something like `ShowPicker = Never | Normal | Force` (the naming is bad, but you get the idea)?

Reply:
Issues 1 through 4 should be resolved now, waiting on comments for 5

Reply:
5 is indeed a problem, but I do not see a good way to resolve it. A visual indicator would be tough to add, and a special hotkey may be another problem for boards with unreliable keyboard drivers. In my opinion, merge and address it only if there is too much trouble.